src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/controller/PrisonerContactController.kt

```diff
@@ -73,11 +78,72 @@ class PrisonerContactController(
     withAddress: Boolean? = true,
   ): List<ContactDto> {
     log.debug("Prisoner: $prisonerId, Type: $contactType, Person: $personId, withAddress = $withAddress")
-    return orderByLastNameAndThenFirstName(contactService.getContactList(prisonerId, contactType, personId, withAddress))
+    return contactService.getContactList(prisonerId, contactType, personId, withAddress)
+      .sortedWith(getDefaultSortOrder())
+  }
+
+  @PreAuthorize("hasRole('PRISONER_CONTACT_REGISTRY')")
+  @GetMapping(PRISON_GET_APRROVED_SOCIAL_CONTACTS_CONTROLLER_PATH)
+  @Operation(
+    summary = "Get Prisoners approved Social Contacts",
+    description = "Returns details of a prisoner's social contacts that have been approved.",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Prisoner Social Contacts Information Returned",
+      ),
+      ApiResponse(
+        responseCode = "400",
+        description = "Incorrect request to Get Prisoner Contacts for Prisoner Identifier",
```

Comment:
Should this description be updated to be unique to the Social Contacts endpoint. It seems the same as the prisoner contact endpoint error description

Reply:
Updated